app/controllers/serial_searches_controller.rb

```diff
@@ -8,9 +8,16 @@ def index
       @title = 'Serial Searches'
       per_page = params[:per_page] || 50
       page = params[:page] || 1
+
       @serial_searches_count = SerialSearch.count
       @ip_addresses_count = IpAddress.count
-      @serial_searches = SerialSearch.order(sort_column + ' ' + sort_direction).page(page).per(per_page)
+
+      if params[:query].present?
+        @serial_searches = SerialSearch.text_search(params[:query]).page(page).per(per_page)
```

Comment:
Since `SerialSearch` returns a scoped result if there isn't a query present, you don't need to wrap this in an if. As an added bonus, you'll be able to order and sort the results still.